site/src/docs/components/login/components.mdx

```diff
@@ -0,0 +1,205 @@
+---
+slug: '/components/login/components'
+title: 'Login - Components'
+---
+
+import { Login } from 'hds-react';
+import PlaygroundPreview from '../../../components/Playground';
+import InternalLink from '../../../components/InternalLink';
+import ExternalLink from '../../../components/ExternalLink';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Components
+
+### LoginProvider
+
+This component creates a React context and initialises all modules. All components and hooks are required to render inside the LoginProvider or React Context will throw an error.
```

Comment:
Like React Contexts in general, it must be positioned so anything using the context is a descendant. Also like other contexts, it will re-render everything inside it when context changes, so it is pretty much up to developers to decide what is enough depth

Reply:
How about regarding the new Header component? This and Header docs could have some linking together. Proper header docs are coming though, we'll see which PR will adapt to the other.

Reply:
Contexts can be nested, if that is what you mean. Devs can use the login inside Header with hooks.

Reply:
Added information about the positioning.